raster/src/main/scala/geotrellis/raster/ConstantTile.scala

```diff
@@ -390,7 +390,7 @@ case class UByteConstantTile(v: Byte, cols: Int, rows: Int,
     *
     * @return  The MutableArrayTile
     */
-  def mutable(): MutableArrayTile = ByteArrayTile.fill(v, cols, rows)
+  def mutable(): MutableArrayTile = UByteArrayTile.fill(v, cols, rows, cellType)
```

Comment:
Good catch!

Reply:
Thanks! I'm a little surprised that wasn't caught sooner.